exchange/bitswap/workers.go

```diff
@@ -9,16 +11,31 @@ import (
 	u "github.com/ipfs/go-ipfs/util"
 )
 
+var TaskWorkerCount = 16
+
+func init() {
+	twc := os.Getenv("IPFS_TASK_WORKERS")
+	if twc != "" {
+		n, err := strconv.Atoi(twc)
+		if err != nil {
+			return
```

Comment:
should print an error here and exit. because the user _meant_ to put something but we're not able to read it. giving feedback would be good. maybe:
```go
if err != nil {
  fmt.Fprintf(os.Stderr, "error: unable to parse IPFS_TASK_WORKERS env var\n")
  fmt.Fprintf(os.Stderr, "%s\n", err)
  fmt.Fprintf(os.Stderr, "IPFS_TASK_WORKERS must be a positive integer", err)
  os.Exit(1)
}
```
(not panic because panics are not nice to end users)